helpers/cryptos/lskBaseCoin.js

```diff
@@ -0,0 +1,230 @@
+const baseCoin = require('./baseCoin');
+const config = require('../../modules/configReader');
+const axios = require('axios');
+const { transactions, cryptography } = require('lisk-sdk');
+const log = require('../log');
+const api = require('../../modules/api');
+const utils = require('../utils');
+const constants = require('../const');
+
+const lskNode = config.node_LSK[0]; // TODO: health check
+const lskService = config.service_LSK[0];
+
+module.exports = class LskBaseCoin extends baseCoin {
+  constructor(token) {
+    super();
+    this.token = token;
+    this.account.keys = api[token.toLowerCase()].keys(config.passPhrase);
+    this.account.network = this.account.keys.network;
+    this.account.keyPair = this.account.keys.keyPair;
+    this.account.address = this.account.keys.address;
+    this.account.privateKey = this.account.keys.privateKey;
+    this.account.addressHex = this.account.keys.addressHex;
```

Comment:
`this.account = Object.assign(this.account, this.account.keys)`?
Or object `this.account.keys` contains additional keys?